### ISSUE TITLE
Font doesn't work on Windows

### ISSUE THREAD

**Issue by rbanffy** [rbanffy/3270font#1]
I've tried it on both Windows XP and Windows 8 and the font installer claims it's not a valid font file.

**Comment by raphaelbastide** [comment]
I had some issue to make it work in CSS on a web page. I achieved to make it work by using quote marks `"font-family:"ibm 3270";`. I don’t know if that can help.

**Comment by rbanffy** [comment]
I had reports that opening a known-good OTF file and pasting all glyphs and metadata works. I'll run a couple tests and, if this method solves the issue (would probably need to base it on a MIT-licensed font, to be on the safe side of licensing) There will be a major change in the font file (which is not a big problem as there are no pending pull requests).

**Comment by DemonCloud** [comment]
I got Same issue~

**Comment by luckyzzzz** [comment]
I like this font,
Hope the issue can be fixed!

**Comment by rbanffy** [comment]
It can be worked around. A friend of mine opened the font in another font editor and saved it. I am investigating what changed, but his version works on Windows. I'll keep this bug open until we find a way to generate the TTF and OTF files in a way Windows likes them.

**Comment by codeman38** [comment]
So I looked into this and think I've figured out what the issue is, after playing around with decompiling and recompiling the font using fonttools.

For whatever reason, Windows appears to choke on fonts if there's an actual character mapped to 0xFFFF in the font's cmap, even though this is technically legal in the TrueType spec. Removing the cmap entry for 0xFFFF in both of the format-4 cmaps was enough to make the font usable.

Probably the easiest fix in production is to just remove uniFFFF from the font entirely-- especially since the glyph is just a blank space anyway.
